training/src/main/scala/com/airbnb/aerosolve/training/AdditiveModelTrainer.scala

```diff
@@ -53,15 +53,19 @@ object AdditiveModelTrainer {
                                    linearFeatureFamilies: java.util.List[String],
                                    priors: Array[String],
                                    nDTreeBuildOptions: NDTreeBuildOptions)
+  def train(sc: SparkContext, input: RDD[Example], config: Config, key: String): AdditiveModel =
+    train(sc, (frac: Double) => input.sample(false, frac), config, key)
```

Comment:
For future reference, we want to sample during each iteration and initialization. We prefer not to double sample during model training.